kanban/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: %i[edit show]
+
+  def index
+    @all_tasks = Task.all
+  end
+
+  def create
+    @task = Task.new(params.require(:task).permit(:name, :description))
+    if @task.save
+      redirect_to tasks_path, notice: 'タスクの登録に成功しました'
+    else
+      flash.now[:alert] = 'タスクの登録に失敗しました'
+    end
+  end
+
+  def new
+    @task = Task.new
+    render :edit
+  end
+
+  def edit; end
+
+  def show; end
+
+  def update
+    @task = Task.find(params[:id])
+    if @task.update params.require(:task).permit(:name, :description)
```

Comment:
```suggestion
    if @task.update(params.require(:task).permit(:name, :description))
```

動作に問題ないですが、引数がある場合はカッコをつけた方がよりGoodです

Reply:
ありがとうございます。カッコつけない方が自然言語っぽくて個人的に好きなんですが、言われてみると、可読性の意味ではつけた方が読みやすくなりそうだなと思いました。